src/scripts/main.js

```diff
@@ -1,3 +1,205 @@
 'use strict';
 
-// write your code here
+const placeColums = [[], [], [], []];
+const buttonPlay = document.querySelector('.button');
+const BASE_CELL_CLASS = 'field-cell';
+const score = document.querySelector('.game-score');
+const gameTable = document.querySelector('.game-header');
+const plaseRow = [...document.querySelector('tbody').children]
+  .map(element => [...element.children]);
+
+for (let i = 0; i < 4; i++) {
+  for (let k = 0; k < 4; k++) {
+    placeColums[k].push(plaseRow[i][k]);
+  }
+}
+
+gameTable.dataset.value = 'off';
+
+function randomInteger(min, max) {
```

Comment:
Consider renaming 'FirtsValue' to 'firstValue' for better readability and proper spelling.